<ui version="4.0" >
 <class>lighteditClass</class>
 <widget class="QDialog" name="lighteditClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>指示灯编辑</string>
  </property>
  <widget class="QPushButton" name="backpb" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title" >
    <string>指示灯设置</string>
   </property>
   <widget class="QLabel" name="label_6" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>230</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>边框颜色</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_biankuangyanse" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_juxingyanse" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>260</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>190</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>边框厚度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_biankuanghoudu" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>260</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>矩形颜色</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>90</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>文字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_wenzicolor" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8" >
    <property name="geometry" >
     <rect>
      <x>240</x>
      <y>120</y>
      <width>54</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text" >
     <string>文字颜色</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="wenzi" >
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>90</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_ziti" >
    <property name="geometry" >
     <rect>
      <x>290</x>
      <y>90</y>
      <width>77</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text" >
     <string>设置字体</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_dangqianzhuangtai" >
    <property name="geometry" >
     <rect>
      <x>90</x>
      <y>30</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>待设状态值</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="le_targetvar" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>10</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>目标变量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_targetvar" >
   <property name="geometry" >
    <rect>
     <x>250</x>
     <y>10</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>54</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>状态数量</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_zhuangtaishuliang" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>40</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
